owrx/source.py

```diff
@@ -378,15 +378,23 @@ def activateProfile(self, profile_id=None):
 
 class RtlSdrSource(SdrSource):
     def getCommand(self):
-        return "rtl_sdr -s {samp_rate} -f {center_freq} -p {ppm} -g {rf_gain} -"
+        command = "rtl_sdr -s {samp_rate} -f {center_freq} -p {ppm} -g {rf_gain}"
+        if self.rtlProps["device"] != "0" :
```

Comment:
This makes `device` key mandatory, and introduces magic value of `"0"` - can we just have it as optional key?